auth-api/src/auth_api/services/org.py

```diff
@@ -270,8 +271,12 @@ def _create_payment_settings(
             payment_account_status = PaymentAccountStatus.CREATED
         elif response.status_code == HTTPStatus.ACCEPTED:
             payment_account_status = PaymentAccountStatus.PENDING
+        elif response.status_code == HTTPStatus.CREATED:
+            payment_account_status = PaymentAccountStatus.FAILED
         else:
             payment_account_status = PaymentAccountStatus.FAILED
+            error_payload = response.json()
```

Comment:
What if the response doesn't include json? and it's a generic exception not a business exception?

Reply:
it's business exception on `def create_org(org_info: dict, user_id)`

Reply:
Removed throwing an exception here and logged an error instead. It breaks many unit tests because an exception was not thrown before.